tests/generation/test_utils.py

```diff
@@ -2730,6 +2730,20 @@ def test_max_length_warning_if_different(self):
                 **model_kwargs,
             )
 
+    def test_max_length_if_input_embeds(self):
+        # PT-only test: TF doesn't have StoppingCriteria
+        article = "Hey, are you conscious?"
```

Comment:
Can we use a different phrase here? Talking about consciousness with these LLMs isn't ideal